### ISSUE TITLE
Be more explicit about the state of the project.

### ISSUE THREAD

**Issue by skx** [skx/dhcp.io#16]
At the moment:
- Updates are capped a little.
  - Partly because of the overhead.
  - Partly because we're using the old API and all changes are heavyweight.
  - When we use the new API #14 we can just use `UPSERT`.
- Account registration(s) are closed.
  - Though I've given a few people new accounts on a one-off-basis.
- We have too many orphans.
  - Need to write a tool to identify accounts that haven't changed IP in _months_.
  - Consider marking such accounts as read-only unless/until they are changed.
- We don't have a good way of contacting users, because emails are option.
  - Disable all acounts without emails?  Until they complain?  Horrid, but workable ..

**Comment by skx** [comment]
I'm going to mark this as closed:

* I closed a bunch of accounts in May.

I disabled all those accounts which hadn't received an update in more than a month, be it manual or via curl.  In short I didn't care that the IP hadn't changed - because static IPs exist - but I did care that an account wasn't used.

I similarly disabled all accounts with either missing or broken emails.  Because they couldn't be authenticated.

Once I made this change a number of users got in touch to complain, and I restored them all.

In short I cleaned up such that I believe I deleted accounts that were orphaned, unused, and obsolete.  Where I made a mistake I rectified it on being informed.

The current state is:

* No "active development".
* The site still works for existing users who .. use it.
* New accounts are disabled, still.

People who want complex things will be pushed towards [commercial DNS hosting](https://dns-api.com/)
